### ISSUE TITLE
requirements.txt modification needed

### ISSUE THREAD

**Issue by ajavamind** [alejandro-ao/langchain-ask-pdf#15]
Thank you Alejandro!
I got chatbot to work on my Windows 11 PC with the following requirement.txt

langchain==0.0.166
PyPDF2==3.0.1
python-dotenv==1.0.0
streamlit==1.18.1
faiss-cpu==1.7.4
altair<5 
openapi==1.1.0
tiktoken

**Comment by puttareddy** [comment]
What was the Python version used for this?

**Comment by ajavamind** [comment]
Python 3.10.11

**Comment by tushaar9027** [comment]
yes this is the right requirements.txt file i added all the packages as well
